src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
```

Comment:
[CODE STYLE] - Avoid using one-letter variable names. Instead of `setQuery` consider using a more descriptive name such as `setSearchQuery`. This will make your code more readable and easier to understand.